src/screens/ProfileScreen/elements/BottomSheetBio.js

```diff
@@ -25,12 +26,16 @@ const BottomSheetBio = React.forwardRef((props, ref) => {
             placeholder="Message prompt"
             keyboardAppearDelay={500}
             editable={!props.isOtherProfile}
+            maxLength={350}
           />
           <Text style={styles.description}>{props.value ? props.value.length : 0}/350</Text>
           {props.error ? <Text style={styles.errorText}>{props.error}</Text> : null}
         </View>
         {!props.isOtherProfile && (
-          <Button style={styles.button} textStyling={styles.textStyling} onPress={props.handleSave}>
+          <Button
+            btnStyle={styles.button}
+            textStyling={styles.textStyling}
+            onPress={props.handleSave}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `maxLength` attribute is added to the `AutoFocusTextArea` component, which is a good practice for limiting user input. However, it's important to ensure that this limit is also enforced on the server side to prevent potential security issues if someone bypasses the client-side check. Also, the change from `style` to `btnStyle` in the `Button` component seems fine as long as it's consistent across all uses of the `Button` component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->